pingdom/src/main/java/com/pulse/controller/AuthenticationController.java

```diff
@@ -0,0 +1,79 @@
+package com.pulse.controller;
+
+import com.pulse.payload.User;
+import com.pulse.repository.AutheneticationRepository;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.text.DateFormat;
+import java.util.Date;
+
+@Path("authentication")
+public class AuthenticationController {
+    AutheneticationRepository autheneticationRepository = new AutheneticationRepository();
+
+    @POST
+    @Path("signup")
+    @Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
```

Comment:
why XML?

Reply:
Initially, I tried with XML format. To understand how it will work.